<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblVideo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbCameraDetails">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>340</width>
      <height>360</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="gbCameraLocationName">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>330</width>
       <height>100</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QComboBox" name="cbCameraLocation">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbName">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbCameraViolations">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>110</y>
       <width>330</width>
       <height>245</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionAdd_camera"/>
   </widget>
   <addaction name="menuCamera"/>
  </widget>
  <action name="actionAdd_camera">
   <property name="text">
    <string>Add camera</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
